lib/p5.js

```diff
@@ -3939,7 +3939,7 @@ amdclean['inputtouch'] = function (require, core) {
       this._setProperty('touchX', touchPos.x);
       this._setProperty('touchY', touchPos.y);
```

Comment:
@kevinb7 That catch is so this part of the code works. We update the position of `touchX` and `touchY` to the last position of lastTouch.

When the update function hits the for loop below, `e.touches` will be empty as you say, so the loop won't run and pull values in from `e.changedTouches`.

An explicit event type check could be made around the `getTouchPos` call above on line 3938.